tutorials/RichMenu.md

```diff
@@ -0,0 +1,78 @@
+
+Like {@tutorial RichDisplay} there is the option to define a template [`MessageEmbed`](https://discord.js.org/#/docs/main/master/class/MessageEmbed) in its constructor, but it will be applied automatically to each page of the menu.
+
+An example of how {@link RichMenu} could be used is in a `help`-like command, this is a simple demo of how it would work:
+
+```javascript
+const menu = new RichMenu(new this.client.methods.Embed()
+	.setColor(0x673AB7)
+	.setAuthor(this.client.user.username, this.client.user.avatarURL())
+	.setTitle('Command List:')
+	.setDescription('Use the arrow reactions to scroll between pages.\nUse number reactions to select an option.')
+);
+
+for (const command of this.client.commands.values()) {
+	menu.addOption(command.name, command.description);
+}
+
+const collector = await menu.run(await msg.send('Loading commands...'));
+
+const choice = await collector.selection;
+if (choice === null) {
+	return collector.message.delete();
+}
+
+const command = this.client.commands.get(menu.options[choice].name);
+
+return collector.message.edit(new this.client.methods.Embed()
+	.setDescription(`The chosen command is \`${command.name}\`.`)
+);
+```
+
+> The code is designed to be placed in a command, inside the `async run(msg)` method but the menu or its options can easily be initialized within the constructor method or the {@link Command.init} method of the command.
+
+## Code Analysis
+
+The creation of the {@link RichMenu} is the same as the one displayed in {@tutorial RichDisplay}, like most of the code and personalization options. Please refer to the {@tutorial RichDisplay} tutorial.
+
+We begin by adding the options, which will be listed in the same order we defined.
+
+```javascript
+for (const command of /* ... */) {
+	menu.addOption(command.name, command.description);
+}
+```
+
+After listing all the options we can call {@link RichMenu.run} on our menu to render it.
+We will store the resulting {@link ReactionHandler} to later access the selected option.
+
+```javascript
+const collector = await menu.run(await msg.send('Loading Commands...'));
+```
+
+We will also need to [`await`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/await) for the user to select an option before continuing:
+
+```javascript
+const choice = await collector.selection;
+```
+
+After obtaining the index of the selected option we can access the option's name and description through our menu:
+
+```javascript
+const command = /* ... */(menu.options[choice].name);
+```
+
+Finally, we show the user the selected command by editing the original [`MessageEmbed`](https://discord.js.org/#/docs/main/master/class/MessageEmbed):
+
+```javascript
+return collector.message.edit(new this.client.methods.Embed()
+	.setDescription(`The chosen command is \`${command.name}\`.`)
```

Comment:
Why not expand this and show full command details (description/extended help/usage/ect).

Reply:
I really only wanted to give a simple practical example (based on yours from the RichMenu PR) that showcased how it worked.
I feel like it would be wrong to enphasize on what could be defined normal javascript (I access a class, I do "stuff" with the class) instead of keeping the RichMenu at the center of the tutorial and avoiding everything extra that does not pertain to it.

Reply:
The example as it stands is too lazy for the quality of tutorial this should be. This requested change will not be dismissed without further address.